css/about.css

```diff
@@ -0,0 +1,167 @@
+/* I tried my best but couldn't go far enough to replicate pixel-for-pixel design */
+
+
+/* My code starts here */
+
+.navigation {
+    margin: 5px;
+    width: 100%;
+    text-align: right;
+    font-size: 20px;
+    padding: 20px;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.container {
+    width: 1000px;
+    margin: 0px auto;
+}
+
+p {
+    padding: 10px auto;
+    font-size: 18px;
+}
+
+h2 {
+    padding: 5px;
+}
+
+h3 {
+    padding: 5px;
+}
+
+.logo {
+    margin: 15px;
+    position: absolute;
+    padding: 10px;
+    left: 0;
+    /* top: 10; */
+    /* display:  */
+}
+
+.jumbo-about {
```

Comment:
not sure if you need to use flexbox here - this class is on your image tag, which does not contain any children. A `margin: auto` would be better here

Reply:
Fixed. Thanks.